styles/main.css

```diff
@@ -309,13 +309,38 @@ td:nth-of-type(7) {
 .burger{
     display:block;
 }
-header nav ul {
-    height:0;
-    overflow:hidden;
-    height:auto;
-}
-.burger~#burger-toggle {
-    left:5px;
-    top:10px;
-    transform:rotate(45deg);
+header nav input { display: none; }
+
+@media screen and ( max-width: 768px ) {
+    header nav ul {
+        height:0;
+        overflow:hidden;
+        padding-top: 0;
+        padding-bottom: 0;
+        margin-top: 1em;
+        margin-bottom: 1em;
+
+        transition: height 500ms ease-in-out;
+    }
+
+    header nav #burger-toggle:checked ~ ul {
+        height: 110px;  
+    }
+    #burger-toggle:checked ~ .burger {
+        transform:rotate(45deg) translate(-5px, 10px);
+    }
+
+    .burger,
+    .burger::before,
+    .burger::after {
+        transition: all 200ms ease-in-out;
+    }
+
+    #burger-toggle:checked ~ .burger::before {
+        opacity: 0;
+    }
+
+    #burger-toggle:checked ~ .burger::after {
+        transform: rotate(-90deg) translate(20px);
+    }

```

Comment:
When the burger-toggle checkbox is checked, hide the middle bar and transform the bottom bar